bip-0352.mediawiki

```diff
@@ -0,0 +1,472 @@
+<pre>
+  BIP: 352
+  Layer: Applications
+  Title: Silent Payments
+  Author: josibake <josibake@protonmail.com>
+          Ruben Somsen <rsomsen@gmail.com>
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0352
+  Status: Draft
+  Type: Informational
```

Comment:
I think this should be Standards Track

Reply:
Per my understanding of BIP2, it's the BIP editor that makes the final designation before merging.